### PR TITLE
Changelog updates

### PR DESCRIPTION
**Pull request by lkysow**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,21 +5,40 @@ FEATURES:
...
+  custom resources then this requires Consul >= 1.9.0 (which is still in beta as of this release).
+
+  To enable, set `controller.enabled: true` in your Helm configuration:
+
+  ```yaml
+  controller:
+    enabled: true
+  ```
+
+  See [https://www.consul.io/docs/k8s/crds](https://www.consul.io/docs/k8s/crds)
```
Comment (lkysow) [inline CHANGELOG.md:23]: WIP: https://github.com/hashicorp/consul/pull/9176

**Review by kschoche — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,21 +5,40 @@ FEATURES:
...
   **It is recommended to enable TLS with this setting enabled because it requires making calls to Consul clients across the cluster.
     Without TLS enabled, these calls could leak ACL tokens should the cluster network become compromised.**
+* Support for custom resource definitions (CRDs) is now generally available.
+  CRDs require Consul >= 1.8.4. If you wish to use `ServiceIntentions`
+  custom resources then this requires Consul >= 1.9.0 (which is still in beta as of this release).
+
+  To enable, set `controller.enabled: true` in your Helm configuration:
+
+  ```yaml
+  controller:
```
Comment (kschoche) [inline CHANGELOG.md:19]: should we mention global here?

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,21 +5,40 @@ FEATURES:
...
   probes will no longer be routable through connect until their Kubernetes health probes are passing.
   Previously, if any connect services were failing their Kubernetes readiness checks they were still routable through connect.
   Users should verify that their connect services are passing Kubernetes readiness probes prior to using health checks synchronization.
+* When health checks are enabled, Consul clients will have `check_update_interval` set to `0s`. Previously,
+  it was set to its default of `5m`. This change ensures the output of the check will show up in the Consul UI immediately. [[GH-674](https://github.com/hashicorp/consul-helm/pull/674)]
+* CRDs: controller default `limits.memory` increased from `30Mi` to `50Mi` and `requests.memory` increased from `20Mi` to `50Mi`
+  based on observed usage. [[GH-649](https://github.com/hashicorp/consul-helm/pull/649)]
+* Using CRDs with an existing cluster may require additional steps to migrate previously created
+  config entries so they can be managed by CRDs. See [https://www.consul.io/docs/k8s/crds/upgrade-to-crds](https://www.consul.io/docs/k8s/crds/upgrade-to-crds)
+  for full details.
```
Comment (ishustava) [inline CHANGELOG.md:38]: is this a breaking change? I'm thinking of a breaking change as a change that will break your workflows without you changing anything in your helm install. Maybe we should just mention it above where we're saying that CRDs are generally available?

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,21 +5,40 @@ FEATURES:
...
-* With the addition of the connect-inject health checks controller any connect services which have failing Kubernetes readiness
+* With the addition of the connect-inject health checks controller, any connect services which have failing Kubernetes readiness
   probes will no longer be routable through connect until their Kubernetes health probes are passing.
   Previously, if any connect services were failing their Kubernetes readiness checks they were still routable through connect.
   Users should verify that their connect services are passing Kubernetes readiness probes prior to using health checks synchronization.
+* When health checks are enabled, Consul clients will have `check_update_interval` set to `0s`. Previously,
+  it was set to its default of `5m`. This change ensures the output of the check will show up in the Consul UI immediately. [[GH-674](https://github.com/hashicorp/consul-helm/pull/674)]
+* CRDs: controller default `limits.memory` increased from `30Mi` to `50Mi` and `requests.memory` increased from `20Mi` to `50Mi`
+  based on observed usage. [[GH-649](https://github.com/hashicorp/consul-helm/pull/649)]
```
Comment (ishustava) [inline CHANGELOG.md:35]: 🤔 is this a breaking change if it's for a beta feature?

**Review by ishustava — APPROVED** [review]
Looks good! I had a couple of questions about what we call a breaking change, but otherwise 👍

File: CHANGELOG.md (hunk 4 above)
Comment (lkysow) [reply CHANGELOG.md:35]: Not sure, I don't mind either way but thought it best to err on the side of breaking change.

File: CHANGELOG.md (hunk 2 above)
Comment (lkysow) [reply CHANGELOG.md:19]: Do you mean like:
```yaml
global:
  name: consul
controller:
  enabled: true
```

`controller` isn't under the `global` key.

File: CHANGELOG.md (hunk 3 above)
Comment (lkysow) [reply CHANGELOG.md:38]: Will move

File: CHANGELOG.md (hunk 4 above)
Comment (ishustava) [reply CHANGELOG.md:37]: 👍 let's keep it then just in case.

File: CHANGELOG.md (hunk 2 above)
Comment (kschoche) [reply CHANGELOG.md:19]: ah okay then nevermind :)

**Pull request merged**